File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    const newArr = new Set(nums)
    const newNewArr = Array.from(newArr)
    for (let i = 0; i < newNewArr.length; i++) {
        nums[i] = newNewArr[i]
    }
    nums.length = newNewArr.length
    return nums.length
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 1
        # compare = nums[0]
        length = 0
        while i < len(nums):
            compare = i - 1
            if nums[compare] != nums[i]:
                length += 1
                nums[length] = nums[i]
            i+=1
        return length + 1